torch/csrc/jit/codegen/cuda/lower_shift.cpp

```diff
@@ -466,6 +223,9 @@ void HaloInfo::build(Fusion* fusion) {
     build(tv->domain());
   }
 
+  if (std::getenv("HALO_INFO")) {
```

Comment:
Can you put this in the utils.cpp debug info options?

Reply:
Ah, sorry, that was my quick debug change. I'll make it available as a debug option.